packages/cli/commands/export/credentials.ts

```diff
@@ -14,10 +14,17 @@ import {
 
 import {
 	Db,
-	GenericHelpers,
```

Comment:
it looks like "logOutput" function from this file is not used anymore so maybe we should delete that

Reply:
Thanks, removed!